application-task-ui/src/main/resources/TaskManager/GanttMacro/Macro.xml

```diff
@@ -1943,8 +1939,16 @@ path=$services.webjars.url('org.webjars.npm:frappe-gantt', "dist/frappe-gantt.mi
   #end
   #if ("$!xcontext.macro.params.projects" != '')
   &lt;p&gt;&lt;b&gt;$escapetool.html($services.localization.render('taskmanager.gantt.macro.projects'))&lt;/b&gt;
+    #set ($translationKeyPrefix = 'TaskManager.TaskManagerClass_project_')
     #foreach ($project in $xcontext.macro.params.projects.split('(?&lt;!\\),'))
-      &lt;span class='xwiki-mention'&gt;$escapetool.html($project.replace('\,',','))&lt;/span&gt;
+      #set ($project = $project.replace('\,',','))
+      #set ($translationKey = $translationKeyPrefix + $project)
+      #if ($services.localization.get($translationKey))
+        #set ($label = $services.localization.render($translationKey))
+      #else
+        #set ($label = $project)
+      #end
+      &lt;span class='xwiki-mention'&gt;$escapetool.html($label)&lt;/span&gt;
```

Comment:
A bit odd to use the `xwiki-mention` class in order to stylize the project. I don't think we should depend on the styling of a separate app. Eventual changes of the mention app  will reflect on our app as well. It also doesn't make sense semantically, having a `xwiki-mention` class added to the span of a `project`.